sdv/constraints/base.py

```diff
@@ -120,13 +121,17 @@ class Constraint(metaclass=ConstraintMeta):
     def _identity(self, table_data):
         return table_data
 
+    def _identity_transform(self, table_data):
```

Comment:
Why not just put in `identity`?

Reply:
`identity` gets used for `self.filter_valid = self._identity` and `self.reverse_transform = self._identity`, cases in which we don't want to run the validation.

Reply:
ok, let's name this more clearly then. Like `_identity_with_validation`